src/api.rs

```diff
@@ -120,6 +120,92 @@ pub unsafe extern "C" fn verify_post(
     raw_ptr(response)
 }
 
+#[allow(dead_code)]
```

Comment:
TODO: Comment this function. If this function expects the caller to provide a u64 for `piece_size` which is equal to the amount of bytes in their piece PLUS bit padding PLUS alignment zeros, say so here.